seqr/views/utils/permissions_utils.py

```diff
@@ -60,8 +89,10 @@ def has_project_permissions(project, user, can_edit=False):
     if can_edit:
         permission_level = CAN_EDIT
 
-    return user.has_perm(permission_level, project) or (user.is_staff and not project.disable_staff_access)\
-        or anvil_has_perm(user, permission_level, project)
+    return user_is_data_manager(user) or \
+           (user_is_analyst(user) and _has_analyst_access(project)) or \
```

Comment:
It seems that it is complicated by using a project_category to enable analysts to access the project. Why don't just keep something like the "disable_staff_access" property for the projects and rename it to "enable_analyst_access"?

Reply:
because this will eventually come from a project group outside seqr